gui: Some gui improved and fixed minor issues.

Croden1999 opened the pull request:
- gui/control_dialog, main_menubar: Restore keyboard controls dialog, the oveylay code has been moved to overlay_dialog.cpp. (Tested on Xiaomi Pad 5 with magnetic keyboard case)
- gui/control_dialog, information_bar, settings, user_management: Add touch scrolling. 
- gui/overlay_dialog: Add lang strings and improve the interface.
- gui/settings_dialog: Fixed missing ua (Ukraine) user language.
- lang: Update lang strings and translation.
- lang: Update id (Indonesia) and ms (Malaysia) translations. - By @Kitakatarashima

![Screenshot_2025-04-07-15-10-26-417_org vita3k emulator](https://github.com/user-attachments/assets/b3b65320-2985-4623-9140-afee36edff88)
![Screenshot_2025-04-07-15-09-37-213_org vita3k emulator](https://github.com/user-attachments/assets/d3e4e08a-f8c4-4653-955d-177bcabc24f5)

https://github.com/user-attachments/assets/477d8894-618f-4c30-862d-aedb197cae94

dima-xd commented:
> * gui/lang: Update lang strings.
> 
> Waiting for Mr. @dima-xd 's PR to merge, and push this PR later.

Mine PR will take time to be completed. Because one thing isn't functioning as it should. So probably I will have to rewrite some Macdu's code

Croden1999 commented:
Ready for review.

Croden1999 committed:
gui: Some gui improved and fixed minor issues.
- gui/control_dialog, main_menubar: Restore keyboard controls dialog, the oveylay code move to overlay_dialog.cpp.
- gui/control_dialog, information_bar, settings, user_management: Add touch scrolling.
- gui/overlay_dialog: Add lang strings and improve the interface.
- gui/settings_dialog: Fixed missing ua (Ukraine) user language.
- lang: Update lang strings and translation.

Croden1999 committed:
lang: Update id (Indonesia) and ms (Malaysia) translation.

Co-authored-by: Kitakatarashima <secretstarsgoldenid@gmail.com>

Croden1999 commented:
Ready for review.

Zangetsu38 commented:
Good works, you can close it here and open pr inside main repo now, just need rebase, have lot conflict and lot things already pressent in master now

Croden1999 commented:
> Good works, you can close it here and open pr inside main repo now, just need rebase, have lot conflict and lot things already pressent in master now

thank you sir.
